bedrock/firefox/views.py

```diff
@@ -795,58 +795,13 @@ def get_template_names(self):
             variation = None
 
         if ftl_file_is_active("firefox/new/desktop") and experience != "basic":
-            template = "firefox/new/desktop/download.html"
+            template = "firefox/download/desktop/download.html"
         else:
-            template = "firefox/new/basic/base_download.html"
+            template = "firefox/download/basic/base_download.html"
 
         return [template]
 
 
-class PlatformViewLinux(L10nTemplateView):
```

Comment:
Can you fill me in on why these went away?

Reply:
The pages for these views have already been removed, this is just some leftover